python/python/lancedb/remote/table.py

```diff
@@ -118,6 +118,22 @@ def create_scalar_index(
 
         return resp
 
+    def create_fts_index(
```

Comment:
actually, can we make this work the same as `AsyncTable` API, which will be our future standard APIs

Reply:
Added